### PR TITLE
Add s390x arch support for k3s

### PR DESCRIPTION
**Pull request by rohitsakala**
### Proposed Changes ####

Add s390x arch support for s390x

#### Types of Changes ####

Updation of dependencies such as klipper helm, klipper lb to new versions which support s390x arch.

#### Verification ####

Github Checks.

Note :- Skipping compact tests for s390x since there is no previous s390x release

#### Linked Issues ####

https://github.com/k3s-io/k3s/issues/3221

### CODE REVIEW

File: scripts/airgap/image-list.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-docker.io/rancher/klipper-helm:v0.6.6-build20211022
-docker.io/rancher/klipper-lb:v0.3.4
+docker.io/rancher/klipper-helm:v0.6.8-build20211123
+docker.io/rancher/klipper-lb:v0.3.5
```
Comment (brandond) [inline scripts/airgap/image-list.txt:2]: this also needs to be updated in the servicelb codebase

File: scripts/airgap/image-list.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 docker.io/rancher/local-path-provisioner:v0.0.21
 docker.io/rancher/mirrored-coredns-coredns:1.8.6
 docker.io/rancher/mirrored-library-busybox:1.34.1
```
Comment (brandond) [inline scripts/airgap/image-list.txt:5]: do we have s390x images for any of the mirrored images? I don't believe our mirroring script would copy those architectures, even if upstream did contain them.

**Review by brandond — CHANGES_REQUESTED** [review]

File: scripts/airgap/image-list.txt (hunk 1 above)
Comment (rohitsakala) [reply scripts/airgap/image-list.txt:2]: Done.

File: scripts/airgap/image-list.txt (hunk 2 above)
Comment (rohitsakala) [reply scripts/airgap/image-list.txt:5]: I can see it for mirror https://hub.docker.com/r/rancher/mirrored-library-busybox/tags. Also, I believe https://github.com/rancher/image-mirror/pull/175 PR should add mirroring for s390x as well.

**Review by briandowns — APPROVED** [review]

**Review by nikolaishields — APPROVED** [review]

**Comment by cwayne18** [comment]
This is dependent on a separate PR for traefik, please don't merge until that is clear

File: scripts/airgap/image-list.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
-docker.io/rancher/klipper-helm:v0.6.6-build20211022
-docker.io/rancher/klipper-lb:v0.3.4
+docker.io/rancher/klipper-helm:v0.6.8-build20211123
```
Comment (brandond) [inline scripts/airgap/image-list.txt:1]: You also need to bump the version of helm-controller in go.mod to a version that uses this as the job image.

**Review by brandond — CHANGES_REQUESTED** [review]

**Comment by raulcabello** [comment]
For Rancher we need k3s [v1.22](https://github.com/rancher/rancher/blob/release/v2.6/Dockerfile.dapper#L1) This PR goes to the master branch. Do we also need to submit another PR to the `release-1.22` branch?

File: scripts/airgap/image-list.txt (hunk 3 above)
Comment (rohitsakala) [reply scripts/airgap/image-list.txt:1]: @brandond Can you review this ? https://github.com/k3s-io/helm-controller/pull/131

Comment (rohitsakala) [reply scripts/airgap/image-list.txt:1]: Done in https://github.com/k3s-io/k3s/pull/5018/commits/2ec0bc4b841ad30c3a7e8a72a3595ee95e0f10ea

**Review by briandowns — APPROVED** [review]

**Comment by dereknola** [comment]
I am merging this despise the Integration test failures, the failure is know flakiness and is being fixed.

**Pull request merged**